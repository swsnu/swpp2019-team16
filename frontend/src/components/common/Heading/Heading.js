import React from 'react';
import PropTypes from 'prop-types';
import Typography from '@material-ui/core/Typography';

HeadingMaterial.propTypes = {
  variant: PropTypes.string,
  title: PropTypes.string.isRequired,
};

function HeadingMaterial({ title }) {
  return (
    <Typography 
      variant="h2" 
      title={title}
    >
    {title}
    </Typography>    
  );
}

export default HeadingMaterial;
