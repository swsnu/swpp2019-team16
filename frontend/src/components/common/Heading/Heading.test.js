import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import Heading from './';

describe('<Heading />', () => {
  it('SHOULD match with snapshot', async () => {
    const { container } = render(<Heading title="Heading title" />);
    expect(container).toMatchSnapshot();
  });
});
