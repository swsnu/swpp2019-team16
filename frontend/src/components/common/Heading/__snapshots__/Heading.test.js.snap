// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> SHOULD match with snapshot 1`] = `
<div>
  <h2
    class="MuiTypography-root MuiTypography-h2"
    title="Heading title"
  >
    Heading title
  </h2>
</div>
`;
